client/js/App.js

```diff
@@ -2,9 +2,15 @@ import React, {Component} from 'react';
 import {View, Text, StatusBar} from 'react-native';
 import Login from './screens/Login';
 import RootStackNavigator from './navigation/RootStackNavigator';
+import Confirmation from './screens/Confirmation';
 
 export default class App extends Component {
   render() {
-    return <RootStackNavigator />;
+    return (
+      <>
+        <Confirmation title="Save the Whales" />
+        <RootStackNavigator />
+      </>
```

Comment:
Your App.js should render RootStackNavigator only. Add Confirmation screen in your navigation so that we can see it.

Reply:
Navigation goes from Campaigns->Campaign Page(Tal's work in progress)->Confirmation so there is no navigation to it right now so it is in App.js so you can see it because on a previous push you commented that you could not see the changes.